### PR TITLE
Include renamed file in manifest

### PR DESCRIPTION
**Pull request by quis**
File was renamed here: https://github.com/alphagov/notifications-utils/commit/6fa9d7e07a5571396b8344d577b2223d8ae444e2#diff-e74407205729a4f75baadfece8ff65da

Won’t install a non-Python file unless it’s in the manifest.

### CODE REVIEW

File: MANIFEST.in [hunk 1]
```diff
@@ -1,6 +1,6 @@
 include notifications_utils/email_preview_template.jinja2
 include notifications_utils/email_template.jinja2
 include notifications_utils/letter_pdf_template.jinja2
-include notifications_utils/letter_preview_template.jinja2
+include notifications_utils/letter_image_template.jinja2
```
Comment (leohemsted) [inline MANIFEST.in]: `include notification_utils/*.jinja2`?

Comment (leohemsted) [inline MANIFEST.in]: or even `recursive-include notification_utils *.jinja2` in case we move the templates into a subdir

Comment (quis) [inline MANIFEST.in]: Cool, didn’t know you could do that. Have changed it.

File: MANIFEST.in [hunk 2]
```diff
@@ -1,6 +1,2 @@
-include notifications_utils/email_preview_template.jinja2
-include notifications_utils/email_template.jinja2
-include notifications_utils/letter_pdf_template.jinja2
-include notifications_utils/letter_preview_template.jinja2
-include notifications_utils/sms_preview_template.jinja2
+recursive-include notification_utils *.jinja2
```
Comment (leohemsted) [inline MANIFEST.in]: sorry, my bad, this should be `notifications_utils`

Comment (leohemsted) [inline MANIFEST.in]: (for reference you can test this by running `python setup.py sdist` and taking a look at the logs it outputs or look in the resultant tar file

`warning: no files found matching '*.jinja2' under directory 'notification_utils'`

Comment (quis) [inline MANIFEST.in]: Squashed again. Running `setup.py` works locally now.

**Pull request merged**